schemas/WebpackOptions.json

```diff
@@ -553,6 +553,11 @@
           "type": "string",
           "minLength": 1
         },
+        "automaticNameMaxLength": {
+          "description": "Sets the max length for the name of a created chunk",
```

Comment:
I would add `(default:100)` to the description in all the options for `automaticNameMaxLength` and  `(default:'~')` for all the options for `automaticNameDelimiter` .  
What say ?  @sokra  
PS: the commit as a whole LGTM 👍